Dockerfile

```diff
@@ -1,15 +1,19 @@
-# To build:
-# $ docker run --rm -v $(pwd):/go/src/github.com/micahhausler/k8s-oidc-helper -w /go/src/github.com/micahhausler/k8s-oidc-helper golang:1.7  go build -v -a -tags netgo -installsuffix netgo -ldflags '-w'
-# $ docker build -t micahhausler/k8s-oidc-helper .
-#
-# To run:
-# $ docker run micahhausler/k8s-oidc-helper
-
-FROM busybox
+FROM golang:1.9
 
 MAINTAINER Micah Hausler, <hausler.m@gmail.com>
 
-COPY k8s-oidc-helper /bin/k8s-oidc-helper
+WORKDIR /build
+ADD . $WORKDIR
+
+RUN go get github.com/ogier/pflag
+RUN go get gopkg.in/yaml.v2
+RUN CGO_ENABLED=0 go build -v -a -tags netgo -installsuffix netgo -ldflags '-w' -o /bin/k8s-oidc-helper .
```

Comment:
I'm not entirely sure this makes sense to do.
 The `-w` turns off DWARF debugging information: you will not be able to use gdb  on the binary to look at specific functions or set breakpoints or get stack traces, because all the metadata gdb needs will not be included.
You will also not be able to use other tools that depend on the information, like pprof profiling.
It saves a tiny amount of space, but you're using a pretty big operating system base image so you won't notice it.